binnacle/lib/DataModel.dart

```diff
@@ -0,0 +1,15 @@
+class DataModel{
+  double windSpeed;
+  double windDirection;
```

Comment:
I know this is pretty late, but i think this class should include members that are of type interfaces to the sensor packages we are using so that they can easily be faked. For example a member of say type ICompass, or I guess Dart uses abstact classes more like interfaces, but essentially we want to be able to fake these sensor packages to better test other things. That's why it might be better to have say compass members rather than the compass heading values. Then we can have functions inside this data model class that starts up listeners via public functions accepting functions/lambda functions as parameters.  Also I need to do more research into our sensor packages as they may already implement some abstract class with their main classes, like the Geolocator class or the compass class. I know this is kind of complex for a comment so we can further discuss over slack or in person.

Reply:
Also, another thing about having these members of the class as doubles/number values, it's forcing redundancy as we have to update a number value in this class when the number in sensor package is changed. If we just had singletons of the individual sensor packages, we could simply reference that value directly.